Vagrantfile

```diff
@@ -10,8 +10,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "ubuntu/trusty64"
-  config.vm.box_version = "20181203.0.1"
+  config.vm.box = "ubuntu/xenial64"
```

Comment:
You decided not to use bento boxes after all?
https://www.vagrantup.com/docs/boxes.html#official-boxes